trello-backup.py

```diff
@@ -1,127 +1,173 @@
-
-	if not TOKEN:
-		print('You need a token to run this app.')
-		print("Visit this url: {0}connect?key={1}&name={2}&response_type=token&expiration={3}".format(API_URL, API_KEY, APP_NAME, TOKEN_EXPIRATION))
-		TOKEN = raw_input('Then enter the token here: ')
-		print('\n[IMPORTANT] Make sure to add the token to the config file.\n')
-		raw_input('Press enter to continue...\n')
-
-	if ORGANIZATION_NAMES and not ORGANIZATION_IDS:
-		ORGANIZATION_IDS = get_organization_ids(ORGANIZATION_NAMES)
-
-	# Parameters to get list of boards
-	boardsPayload = {
-		'key':API_KEY,
-		'token':TOKEN,
-		'filter':'open',
-		'lists':'open',
-	}
-	# Parameters to get board contents
-	boardPayload = {
-		'key':API_KEY,
-		'token':TOKEN,
-		'lists':'open',
-		'fields':'all',
-		'actions':'all',
-		'action_fields':'all',
-		'actions_limit':'1000',
-		'cards':'all',
-		'card_fields':'all',
-		'card_attachments':'true',
-		'lists':'all',
-		'list_fields':'all',
-		'members':'all',
-		'member_fields':'all',
-		'checklists':'all',
-		'checklist_fields':'all',
-		'organization':'false',
-	}
-	boards = requests.get(API_URL + "members/my/boards", data=boardsPayload)
-	try:
-		if len(boards.json()) <= 0:
-			print('No boards found.')
-			return
-	except ValueError:
-		print('Unable to access your boards. Check your key and token.')
-		return
-	if not os.path.exists(OUTPUT_DIRECTORY):
-		os.makedirs(OUTPUT_DIRECTORY)
-
-	print('Backing up boards:')
-	epoch_time = str(int(time.time()))
-
-	for board in boards.json():
-		if ORGANIZATION_IDS and (not board["idOrganization"] or not board["idOrganization"] in ORGANIZATION_IDS):
-			continue
-
-		print(u"    - {0} ({1})".format(board["name"], board["id"]))
-		boardContents = requests.get(API_URL + "boards/" + board["id"], data=boardPayload)
-		with io.open(OUTPUT_DIRECTORY + u'/{0}_'.format(board["name"].replace("/","-")) + epoch_time + '.json', 'w', encoding='utf8') as file:
-			args = dict( sort_keys=True, indent=4) if PRETTY_PRINT else dict()
-			data = json.dumps(boardContents.json(), ensure_ascii=False, **args)
-			file.write(unicode(data))
-
-def get_organization_ids(ORGANIZATION_NAMES):
-	selected_organizations = []
-	for organization in ORGANIZATION_NAMES.split(','):
-		selected_organizations.append(organization.strip())
-	organization_ids = []
-
-	# Parameters to get a list of organizations
-	organizationsPayload = {
-		'key':API_KEY,
-		'token':TOKEN,
-	}
-
-	organizations = requests.get(API_URL + "members/my/organizations", data=organizationsPayload)
-	if len(organizations.json()) <= 0:
-		print('No organizations found.')
-	else:
-		for organization in organizations.json():
-			if organization["name"] in selected_organizations:
-				organization_ids.append(organization["id"])
-	return organization_ids
-
-
-if __name__ == '__main__':
-	main()
+import logging
+import argparse
+
+api_url = ""
+ORGANIZATION_NAMES = ""
+ORGANIZATION_IDS = []
+
+
+def get_organization_ids(ORGANIZATION_NAMES, api_key, token):
```

Comment:
Its somewhat confusing that the `ORGANIZATION_NAMES` of this function shadows the globally defined variable.  I think it should have a local scope, or don't bother to pass it at all because its already in the global scope?